.github/workflows/ci_pipeline.yml

```diff
@@ -110,47 +97,41 @@ jobs:
           - Second_orderSDC
           - soft_failure
           - TOMS
-        python: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        # python: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        python: ['3.10']
```

Comment:
Same as above

Reply:
Done in b460c3bfbe4709e718938daa18ea4d28386c3158